.jshintrc

```diff
@@ -1,5 +1,5 @@
 {
-  "globals"     : { "Phaser": false, "PIXI": false, "p2": false, "CocoonJS": false, "process": false, "game": true },
+  "globals"     : { "Phaser": false, "PIXI": false, "p2": false, "CocoonJS": false, "process": false, "game": true, "randomTexture": true, "smoothstep": true },
```

Comment:
I think this should be "randomTexture": false, "smoothstep": false, the boolean value represents whether or not you can write to this variable. And in the case of functions you don't want to accidentally write to them.